images/li/sle12_sp3/config.sh

```diff
@@ -38,13 +38,6 @@ popd
 #--------------------------------------
 suseImportBuildKey
 
-#=========================================
-# Delete persistent-net.rules
-#-----------------------------------------
-# Avoid recreation of interface names at each boot
-# This goes along with net.ifnames=1
-rm -f /etc/udev/rules.d/70-persistent-net.rules
```

Comment:
This will break systems that get re-mapped to a different blade, which if I remember correctly was a use case we needed to support.

Reply:
To my understanding this is safe if only one link policy is active as it is with

[Link]
NamePolicy=mac
MACAddressPolicy=persistent

Can you explain more detailed why you think 70-persistent-net.rules will break the system if the blade moves ? I can't remember and as this was all done in files on obs at that time the history is not helpful. Thanks

Reply:
What I know for sure is if you delete 70-persistent-net.rules the link NamePolicy setup in systemd has no effect anymore. So this is the place where the policy gets applied. Our static udev rules before enforces the policy. This change basically sets only one policy (mac) and keeps that consistent across the system. According to Marius that's the better way and interfaces gets created also only once

Reply:
The creation of the ifcfg-* files is the problem. On a different blade an interface may have the same name but a different MAC. This then triggers a conflict in the naming scheme since the existing ifcfg- file will not be clobbered with the same name and the new MAC data. What I do not remember is if in the case of the LI/VLI implementation the MAC ends up being written into the ifcfg- file or not. If it does then that will triger an issue when the system is re-mapped to a different blade.

Reply:
> the MAC ends up being written into the ifcfg-

This is not the case. I checked the code in azure_li_services/network.py and we don't write the MAC to the file. So I think this should not be an issue